google-cloud-core/src/main/java/com/google/cloud/GrpcServiceOptions.java

```diff
@@ -226,7 +226,7 @@ protected GrpcServiceOptions(
   /**
    * Returns a builder for API call settings.
    */
-  protected ApiCallSettings.Builder apiCallSettings() {
+  protected UnaryApiCallSettings.Builder apiCallSettings() {
```

Comment:
> it's not clear which class you are referencing

Sorry, but you got it right I was suggesting renaming `PageStreamingCallSettings`.
Proposed naming sounds much better to me. I would also consider `PagedCallSettings`, any reason you ruled that out?